dataline-api/build.gradle

```diff
@@ -75,7 +74,7 @@ dependencies {
 sourceSets {
     main {
         java {
-            srcDirs "$buildDir/generated/src/gen/java", "$buildDir/generated/src/main/java", "$projectDir/src/main/java"
+            srcDirs "$buildDir/generated/api/server/src/gen/java", "$buildDir/generated/api/client/src/main/java", "$projectDir/src/main/java"
```

Comment:
this appears to be necessary otherwise gradle up-to-date gets confused since there are two different tasks writing to the same dir.